service/src/main/java/bio/terra/externalcreds/services/PassportProviderService.java

```diff
@@ -206,14 +202,7 @@ void authAndRefreshPassport(LinkedAccount linkedAccount) {
   private LinkedAccountWithPassportAndVisas getRefreshedPassportsAndVisas(
       LinkedAccount linkedAccount) {
     var clientRegistration =
-        providerOAuthClientCache
-            .getProviderClient(linkedAccount.getProviderName())
-            .orElseThrow(
-                () ->
-                    new ExternalCredsException(
-                        String.format(
-                            "Unable to find configs for the provider: %s",
-                            linkedAccount.getProviderName())));
```

Comment:
Did this error get moved elsewhere?

Reply:
Yup! Since `provider` is now a type-safe enum, we only have to error if we don't have the key in the map of provider configs. So, I made a convenience method to check for key existence in the map, and migrated all calls to use that.

https://github.com/DataBiosphere/terra-external-credentials-manager/pull/176/files